src/signs/ui/display/SignInformation.test.tsx

```diff
@@ -87,6 +95,15 @@ describe('Sign Information', () => {
     })
     expectWordPropertiesToBeInTheDocument(wordErimmatu)
     expectWordPropertiesToBeInTheDocument(wordLipu)
+
+    expect(screen.getAllByRole('link')[0]).toHaveAttribute(
+      'href',
+      `/dictionary/${wordErimmatu._id}`
+    )
+    expect(screen.getAllByRole('link')[1]).toHaveAttribute(
+      'href',
+      `/dictionary/${wordLipu._id}`
+    )
```

Comment:
This is brittle because it assumes a certain order of links. Adding an unrelated link to the component might break the test. Use `name`. If it is still not unique see try [querying within element](https://testing-library.com/docs/dom-testing-library/api-within).